app/src/components/modals/ScoutViewer.js

```diff
@@ -558,38 +559,25 @@ function ScoutViewer({
                       />
                     </View>
                   )}
-                  {field.type === 'checkbox' && (
-                    <Checkbox
-                      title={''}
-                      disabled={!editingActive}
-                      colors={colors}
-                      options={field.options}
-                      value={
-                        tempData[index] &&
-                        tempData[index] != null &&
-                        tempData[index] !== ''
-                          ? Object.values(tempData[index]).map(value => {
-                              return Number.parseInt(value, 10);
-                            })
-                          : []
-                      }
-                      onValueChange={value => {
-                        console.log('new checkbox value: ' + value);
-                        // print out the values of the dictionary tempData[index]
-
-                        console.log(
-                          'values of temp data index: ' +
-                            Object.values(tempData[index]),
-                        );
-                        console.log(
-                          'tostring of temp data index: ' +
-                            tempData[index].toString(),
-                        );
-                      }}
-                    />
+                  {field.type === 'checkboxes' && (
+                    <View>
+                      <Checkbox
+                        title={field.question}
+                        options={field.options}
+                        value={tempData[index]}
+                        disabled={!editingActive}
+                        colors={colors}
+                        editingActive={editingActive}
+                        onValueChange={value => {
```

Comment:
When testing this feature, it did not correctly save the different checkboxes for me.

Reply:
it works on my computer